test/e2e/storage/vsphere_volume_vsan_policy.go

```diff
@@ -181,21 +194,72 @@ var _ = framework.KubeDescribe("VSAN policy support for dynamic provisioning [Vo
 		scParameters[Policy_ObjectSpaceReservation] = ObjectSpaceReservationCapabilityVal
 		scParameters[Datastore] = VmfsDatastore
 		framework.Logf("Invoking Test for VSAN storage capabilities: %+v", scParameters)
-		err := invokeInvalidVSANPolicyTestNeg(client, namespace, scParameters)
+		err := invokeInvalidPolicyTestNeg(client, namespace, scParameters)
 		// This will make sure err is always returned.
 		Expect(err).To(HaveOccurred())
 		errorMsg := "The specified datastore: \\\"" + VmfsDatastore + "\\\" is not a VSAN datastore. " +
 			"The policy parameters will work only with VSAN Datastore."
-		framework.Logf("errorMsg: %+q", errorMsg)
+		if !strings.Contains(err.Error(), errorMsg) {
+			Expect(err).NotTo(HaveOccurred())
+		}
+	})
+
+	It("verify an existing and compatible SPBM policy is honored for dynamically provisioned pvc using storageclass", func() {
+		By(fmt.Sprintf("Invoking Test for SPBM policy: %s", os.Getenv("VSPHERE_SPBM_GOLD_POLICY")))
```

Comment:
OK. I assume there are changes you need to cause these vars to be set in the CI test environment?